perf/benchmark/graph_plotter/graph_plotter.py

```diff
@@ -83,7 +83,7 @@ def get_constructed_query_str(args):
     if args.x_axis == "qps":
         return 'ActualQPS==@ql and ' + args.query_str + ' and Labels.str.endswith(@telemetry_mode)'
     elif args.x_axis == "conn":
-        return args.query_str + ' and NumThreads==@ql and Labels.str.endswith(@telemetry_mode)'
+        return 'NumThreads==@ql and Labels.str.endswith(@telemetry_mode)' in cases where the ActualQPS is not 1000 
```

Comment:
this doesn't seem syntactically correct? is it missing a `#`, or am I out of touch with modern python??

Reply:
sorry cut and paste error